src/main/java/step3/model/Movement.java

```diff
@@ -0,0 +1,23 @@
+package step3.model;
+
+public class Movement {
+    private static final int DEFAULT_COUNT = 0;
+    private static final int FORWARD = 1;
+    private int moveCount;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 어떨까요?
```suggestion
    private static final int FORWARD = 1;

    private int moveCount;
```